cmd/wallet-js-sdk/test/specs/didcomm/connect.v2.spec.js

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import {expect} from "chai";
+import {
+    loadFrameworks,
+    testConfig,
+    DIDEXCHANGE_STATE_TOPIC,
+    POST_STATE,
+    DIDEXCHANGE_STATE_REQUESTED,
+    wait
+} from "../common";
+import {
+    connectToMediator,
+    createWalletProfile,
+    DIDComm, DIDManager,
+    getMediatorConnections,
+    UniversalWallet,
+    waitForEvent
+} from "../../../src";
+
+const ALICE_LABEL_V2 = "alice-agent-v2"
+const BOB_LABEL_V2 = "bob-agent-v2"
+
+let alice, bob
+
+before(async function () {
+    alice = await loadFrameworks({name: ALICE_LABEL_V2})
```

Comment:
you are still using DIDCommV1 here? I see you haven't passed any `mediaTypeProfiles`

Reply:
i'm not sure where the wasm agent needs to be updated, adding options here doesn't affect the wasm agent in any way (agent remains with default mediaTypeProfile even if i pass didcomm/v2)

Reply:
then it has to be fixed to test DIDCommV2 features

Reply:
fixed in aries-js-worker